Builds/VisualStudio/stellar-core.vcxproj

```diff
@@ -546,7 +546,7 @@ exit /b 0
     <ClCompile Include="..\..\src\util\test\SchedulerTests.cpp" />
     <ClCompile Include="..\..\src\util\XDRCereal.cpp" />
     <ClCompile Include="..\..\src\util\xdrquery\test\XDRQueryTests.cpp" />
-    <ClCompile Include="..\..\src\util\xdrquery\XDRMatcher.cpp" />
```

Comment:
I think this commit should probably be the first in your PR as the other commits seem to depend on the new name

Reply:
I did the rename after all the changes, but I guess this got messed up when I was squashing some intermediate commits. Updated the history.

Reply:
It looks like you need to refresh timestamps for commits as to not confuse Github: right now that commit looks like the last one in the list but it's the first one in your chain.

The way to do this, is to rebase by running `git rebase --force-rebase --exec "sleep 2" --exec "git commit --amend --date=now" master`

Reply:
The rename is supposed to be the last commit and it seems like it is both in git hub and git logs.

Reply:
ah yes, I saw that 🤦‍♂️ . In my mind you were going to rename first before making the other changes. Looks good